vnext/Microsoft.ReactNative/Modules/NetworkingModule.cpp

```diff
@@ -229,11 +229,16 @@ void NetworkingModule::NetworkingHelper::OnRequestError(int64_t requestId, std::
 void AttachContentHeaders(
     winrt::Windows::Web::Http::IHttpContent content,
     winrt::Windows::Web::Http::Headers::HttpMediaTypeHeaderValue contentType,
-    std::string contentEncoding) {
+    std::string contentEncoding,
+    std::string contentLength) {
```

Comment:
you could parse the value earlier and pass it here as either `uint64_t` with a sentinel value like (uint64_t)-1 to denote "unspecified", or as a `std::optional<uint64_t>`

Reply:
makes sense, but I've seen that `ContentEncoding` is using `ParseAdd`, so I thought that would keep parsing to the same method. But I can move the parsing if needed.